src/devices_models/devices/common/add_to_expression.jl

```diff
@@ -15,16 +15,34 @@ function add_expressions!(
     return
 end
 
+function add_expressions!(
+    container::OptimizationContainer,
+    ::Type{T},
+    devices::U,
+    model::ServiceModel{V, W};
+    meta = CONTAINER_KEY_EMPTY_META,
```

Comment:
You shouldn't need to pass meta here. The meta field should be determined inside of the function calling `add_expression_container!`